main.nf

```diff
@@ -44,6 +44,8 @@ def helpMessage() {
      --known_splices [file]             Supply a .gtf file containing known splice sites (bismark_hisat only)
      --slamseq [bool]                   Run bismark in SLAM-seq mode
      --local_alignment [bool]           Allow soft-clipping of reads (potentially useful for single-cell experiments)
+     --bismark_align_cpu_per_multicore [int]     Specify how many CPUs are required per --multicore for bismark align (default = 3)
+     --bismark_align_mem_per_multicore [memory]  Specify how much memory is required per --multicore for bismark align (default = 13.GB)
```

Comment:
Should we be using `[memory]` here? I think its best that we stick to the standard object types so in this case it would be `[str]`. The default value should help users figure out how to specify it.

Reply:
Done in https://github.com/nf-core/methylseq/pull/137/